docs/resources/certificate_authority.md

```diff
@@ -0,0 +1,223 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "AAA"
+layout: "aci"
+page_title: "ACI: aci_certificate_authority"
+sidebar_current: "docs-aci-resource-aci_certificate_authority"
+description: |-
+  Manages ACI Certificate Authority
+---
+
+# aci_certificate_authority #
+
+Manages ACI Certificate Authority
+
+
+
+## API Information ##
+
+* Class: [pkiTP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/pkiTP/overview)
+
+* Supported in ACI versions: 1.0(1e) and later.
+
+* Distinguished Name Formats:
+  - `uni/tn-{name}/certstore/tp-{name}`
+  - `uni/userext/pkiext/tp-{name}`
```

Comment:
should we switch the dn formats? this way the import statement below would point to the non-cloud apic as example

Reply:
ok